packages/hooks/src/test/index.test.js

```diff
@@ -514,16 +514,16 @@ test( 'Verify doingFilter, didFilter and hasFilter.', () => {
 	expect( test ).toBe( 'someValue' );
 	expect( filterCalls ).toBe( 1 );
 	expect( didFilter( 'runtest.filter' ) ).toBe( 1 );
-	expect( hasFilter( 'runtest.filter' ) ).toBe( 1 );
-	expect( hasFilter( 'notatest.filter' ) ).toBe( 0 );
+	expect( hasFilter( 'runtest.filter' ) ).toBe( true );
+	expect( hasFilter( 'notatest.filter' ) ).toBe( false );
 	expect( doingFilter() ).toBe( false );
 	expect( doingFilter( 'runtest.filter' ) ).toBe( false );
 	expect( doingFilter( 'notatest.filter' ) ).toBe( false );
 	expect( currentFilter() ).toBe( null );
 
 	expect( removeAllFilters( 'runtest.filter' ) ).toBe( 1 );
 
-	expect( hasFilter( 'runtest.filter' ) ).toBe( 0 );
+	expect( hasFilter( 'runtest.filter' ) ).toBe( true );
 	expect( didFilter( 'runtest.filter' ) ).toBe( 1 );
```

Comment:
Unrelated, but I'm guessing returning a boolean for `didFilter` sounds like a good candidate for a follow-up pull request.